core/src/main/java/org/jobrunr/jobs/context/JobDashboardLogger.java

```diff
@@ -86,6 +91,12 @@ public JobDashboardLogLine(Level level, String logMessage) {
             this.logMessage = logMessage;
         }
 
+        public JobDashboardLogLine(Level level, Instant logInstant, String logMessage) {
+            this.level = level;
+            this.logInstant = logInstant;
```

Comment:
`logInstant` is set to `Instant.now()` in the other constructor, other `JsonMapper`s use reflection hacks to override it

Reply:
Makes sense. Let's have the other constructor call this constructor however to minimize code.